rest/rest-resources/src/main/java/org/apache/brooklyn/rest/resources/AbstractBrooklynRestResource.java

```diff
@@ -143,11 +145,12 @@ public Object resolve() {
         
         private static Object UNRESOLVED = "UNRESOLVED".toCharArray();
         
-        private static Object getImmediateValue(Object value, @Nullable Entity context, @Nullable Duration timeout) {
+        private static Object getImmediateValue(Object value, @Nullable Entity context, @Nullable Boolean immediately, @Nullable Duration timeout) {
             return Tasks.resolving(value)
                     .as(Object.class)
                     .defaultValue(UNRESOLVED)
                     .timeout(timeout)
```

Comment:
What happens with a timeout _and_ an immediately? Should this be `.timeout( immediately ? 0 : timeout)`? (with adjustments for Boolean)